Dockerfile

```diff
@@ -1,16 +1,17 @@
-FROM ruby:2.6
+FROM registry.access.redhat.com/ubi8/ruby-26
 
-WORKDIR /app
+USER root
 
-RUN apt-get update && apt-get install -y qt5-default libqt5webkit5-dev \
-      gstreamer1.0-plugins-base gstreamer1.0-tools gstreamer1.0-x libopenscap-dev \
-      postgresql-client
+COPY . /tmp/src
```

Comment:
We should probably only copy what is needed. For example, I sometimes have SCAP scans or DB backups or other large files in my project root that don't need to be a part of the image.